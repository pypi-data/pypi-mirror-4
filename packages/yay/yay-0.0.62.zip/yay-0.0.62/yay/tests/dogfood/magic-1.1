foo: magic-1.2

