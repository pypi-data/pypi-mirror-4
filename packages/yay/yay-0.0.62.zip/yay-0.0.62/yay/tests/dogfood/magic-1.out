bar: its a kind of magic
foo: magic-1.2
